freebasics/templates/comments/preview.html

```diff
@@ -1,28 +1,33 @@
 {% extends "base.html" %}
 {% load wagtailcore_tags wagtailimages_tags comments mptt_tags molo_commenting_tags i18n %}
 {% get_comments_content_object for form as self %}
+
 {% block breadcrumbs %}
-<div class="breadcrumbs fb-body">
-  <span><a class="fb-accent-2" href="/">{% trans "Home" %}</a></span> &gt;
-  <span class="active">{{ self.title }}</span>
-</div>
+  <div class="breadcrumbs">
+    <a class="breadcrumbs__anchor" href="/">{% trans "Home" %}</a>
+    <span class="breadcrumbs__anchor-is-active">{{ self.title }}</span>
+  </div>
 {% endblock %}
 
 {% block content %}
-<div class="block-category fb-body fb-card {self.get_effective_extra_style_hints}}">
-  {% with parent_section=self.get_parent_section ancestor=self.get_parent_section.get_ancestors.last %}
-    <div class="icon">
-      {% if self.image %}
-        {% image self.image width-98 %}
-      {% elif parent_section.image %}
-        {% image parent_section.image width-98 %}
-      {% else %}
-        {% image ancestor.sectionpage.image width-98 %}
-      {% endif %}
-    </div>
-    <h6>{{ancestor.sectionpage.title}}</h6>
-    <h6 class="block-heading fb-accent-1">{{parent_section.title}}</h6>
-  {% endwith %}
+<!-- What does this page do? -->
```

Comment:
If the user didn't write any comment but tries to submit a comment, it will take them to this page and show them an error asking to write a comment.